Database/stylesdb.js

```diff
@@ -0,0 +1,46 @@
+const fs = require('fs')
+if(fs.existsSync('.env')){
+  require('dotenv').config()
+}
+
+const connectingString = process.env.DATABASE_URL
+
+const pgp = require('pg-promise')()
+
+const db = pgp(connectingString)
+
+const CREATE_STYLES = `INSERT INTO styles
+VALUES($1,$2,$3)`
+
+const VIEW_STYLES =`SELECT  styles.style_image, stylist.name AS stylist_name
+FROM customers
+JOIN appointments
+ON customers.id = appointments.id
+JOIN stylist
+ON appointments.stylist_id = stylist.id
+JOIN styles
+ON appointments.style_id = styles.id
+WHERE active = 'true'`
+
+const UPDATE_STYLES =`UPDATE styles
+SET Style_name = $1,style_image = $2 ',style_duration = $3
+WHERE ID = $4;`
+
+const DELETE_STLYES = `DELETE FROM styles
+WHERE id = $1`
+
+const styles ={
+  createStyles:(Style_name,Style_image,Style_duration) => {
```

Comment:
It is idiomatic Javascript to name variables starting with a lower case letter, and use camelCase instead of underscores to separate words in the variable name.  These should be `styleName`, `styleImage`, `styleDuration`.  As well, consider the use of whitespace (between the semicolon and value) for clarity.